src/components/manage/Controlpanels/Controlpanels.jsx

```diff
@@ -167,6 +167,11 @@ class Controlpanels extends Component {
           group: this.props.intl.formatMessage(messages.usersandgroups),
           title: this.props.intl.formatMessage(messages.users),
         },
+        {
+          '@id': '/usergroupmembership',
+          group: 'Users and Groups',
+          title: 'User Group Membership', // this.props.intl.formatMessage(messages.users),
```

Comment:
Can we re-add the i18n config? We have an undergoing PR trying to fix some of the other CP translations already: https://github.com/plone/volto/pull/3486/files#diff-9d05cfbd5917826f55a5865e35ea15138f8054bf5d66f95192151af7586cdd1d

Reply:
oops yes, of course. Will do today.
Thanks for taking time and reviewing!

Reply:
Thanks for hte hint! Done in https://github.com/plone/volto/pull/3349/commits/70ce484e65deadc0865357bee8de0caaacdadfe5